### ISSUE TITLE
Sometimes, site search result links not working

### ISSUE THREAD

**Issue by joshbuker** [MrGreensWorkshop/MrGreen-JekyllTheme#70]
**Describe the bug**

When using the site search functionality, sometimes clicking on posts does not actually do anything. Instead it closes the search bar with no action.

**Screenshots**
<img src="https://user-images.githubusercontent.com/1802598/176817357-3fc027a9-ca21-4483-9d42-cb8fb6d80662.png" width="600">

**Expected behavior**

When clicking on one of the search results, the user is taken to the respective page.

**To Reproduce**
1. Visit https://joshbuker.github.io/
2. Search for a post, e.g. "sample" at time of writing.
3. Click on one of the results
4. Instead of taking you to the result, it closes the search with no action.

**Desktop (please complete the following information):**
- OS: Linux
- Browser: Firefox

**Mobile devices (please complete the following information):**

**Additional context**
I got it to actually follow through once, but couldn't replicate it consistently. Not sure what exactly is causing it to glitch out.

**Comment by MrGreensWorkshop** [comment]
It's not reproducible. I will keep this open for a while because I saw once or twice.

**Comment by MrGreensWorkshop** [comment]
I think I found why it's looks like that way.
<img width="1305" alt="Screen Shot 2022-07-03 at 12 00 55" src="https://user-images.githubusercontent.com/76992231/177022792-ee04a775-f4f3-4017-93e9-d7e513e266f4.png">

A tag shorter than li tag, so if you click the space between page title and date search results will be closed. its expected behavior.

**Comment by MrGreensWorkshop** [comment]
Removing the `width: 73%` will fix the problem. I will check if is there any layout problems with different screen sizes.

<img width="1132" alt="Screen Shot 2022-07-03 at 13 29 15" src="https://user-images.githubusercontent.com/76992231/177024676-3f5aefd9-2d72-4622-b689-fcfada4fab45.png">

**Comment by MrGreensWorkshop** [comment]
if you able to click the bars between links, search results will be closed. its expected behavior.

**Comment by MrGreensWorkshop** [comment]
when its removed, page titles and dates overlaps..
<img width="613" alt="Screen Shot 2022-07-03 at 14 02 52" src="https://user-images.githubusercontent.com/76992231/177025579-36ca5e75-cbfd-439c-bb22-a2e814d70093.png">

**Comment by MrGreensWorkshop** [comment]
Changing search result html and adding float left and right fix the problem permanently.

```
#search-results > li .date {
  float: right;
}

#search-results > li .title {
  float: left;
}

```
<img width="1055" alt="Screen Shot 2022-07-04 at 11 24 05" src="https://user-images.githubusercontent.com/76992231/177083714-b20f8c9b-3845-4220-95fc-1153e2f0da6d.png">

**Comment by MrGreensWorkshop** [comment]
fixed with #77
